fe/fe-core/src/main/java/com/starrocks/http/meta/MetaService.java

```diff
@@ -256,7 +256,10 @@ public void executeGet(BaseRequest request, BaseResponse response) {
                 return;
             }
 
-            GlobalStateMgr.getCurrentState().setImageJournalId(version);
+            // When subDirStr is empty, image belongs to LocalMetaStore and others belong to staros.
+            if (Strings.isNullOrEmpty(subDirStr)) {
```

Comment:
Please create a new variable which stands for local meta rather than using this implicit logic

Reply:
done